translation_center.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", ">= 3.1.0", "<= 3.2.12"
+  s.add_dependency "rails", ">= 3.1.0", "<= 3.2.40"
```

Comment:
We will have to update this every new version?
Wouldn't doing the following be better?
```
s.add_dependency "rails", ">= 3.1.0", "~> 3.2"‎
```